// diff -Naur FreeRDP-Remmina-356c033.orig/CMakeLists.txt FreeRDP-Remmina-356c033/CMakeLists.txt
--- FreeRDP-Remmina-356c033.orig/CMakeLists.txt	2012-02-10 19:54:23.000000000 +0100
+++ FreeRDP-Remmina-356c033/CMakeLists.txt	2013-04-26 18:03:15.603759100 +0200
@@ -64,11 +64,6 @@
 endif()
 
 check_include_files(sys/param.h HAVE_SYS_PARAM_H)
-check_include_files(sys/socket.h HAVE_SYS_SOCKET_H)
-check_include_files(arpa/inet.h HAVE_ARPA_INET_H)
-check_include_files(netinet/in.h HAVE_NETINET_IN_H)
-check_include_files(termios.h HAVE_TERMIOS_H)
-check_include_files(netdb.h HAVE_NETDB_H)
 check_include_files(fcntl.h HAVE_FCNTL_H)
 check_include_files(unistd.h HAVE_UNISTD_H)
 check_include_files(sys/un.h HAVE_SYS_UN_H)
@@ -87,24 +82,9 @@
 	add_definitions(-DHAVE_PTHREAD)
 endif()
 
-find_suggested_package(GCRYPT)
-if(GCRYPT_FOUND)
-	add_definitions(-DHAVE_LIBGCRYPT)
-endif()
-
 find_required_package(GTK)
 if(GTK_FOUND)
-	add_subdirectory(remmina)
-	add_subdirectory(remmina-plugins)
-	add_subdirectory(remmina-plugins-gnome)
-endif()
-
-find_suggested_package(AVAHI)
-if(AVAHI_FOUND)
-	add_definitions(-DHAVE_LIBAVAHI_UI)
-	add_definitions(-DHAVE_LIBAVAHI_CLIENT)
+	add_subdirectory(remmina-plugin-open)
 endif()
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/config.h.in ${CMAKE_CURRENT_SOURCE_DIR}/config.h)
-
-install(DIRECTORY include/remmina DESTINATION include/remmina FILES_MATCHING PATTERN "*.h")
